ipaserver/install/ipa_replica_prepare.py

```diff
@@ -448,12 +450,23 @@ def copy_pkinit_certificate(self):
     def copy_misc_files(self):
         self.log.info("Copying additional files")
 
-        self.copy_info_file(CACERT, "ca.crt")
         cacert_filename = paths.CACERT_PEM
         if ipautil.file_exists(cacert_filename):
             self.copy_info_file(cacert_filename, "cacert.pem")
         self.copy_info_file(paths.IPA_DEFAULT_CONF, "default.conf")
 
+    def retrieve_ca_certs(self):
+        self.log.info("Retrieving CA certificates")
+        ca_certs = certstore.get_ca_certs(api.Backend.ldap2, api.env.basedn,
+                                          api.env.realm, False)
+        certs = (c[0] for c in ca_certs if c[2] is not False)
+        dest = os.path.join(self.dir, "ca.crt")
+        try:
+            x509.write_certificate_list(certs, dest)
+        except Exception as e:
+            raise admintool.ScriptError(
+                "File creation {0} failed: {1}".format(dest, e))
+
```

Comment:
This looks a lot like `install_ca_cert()` from replicainstall.py. This function changed a bit (but not a whole lot) during installer-refactoring (see https://github.com/jcholast/freeipa/tree/installer-refactoring if you want). I would propose to wait for it to get pushed (this Friday) and then rather reuse its code instead of having it copy pasted here :)